src/components/Components.tsx

```diff
@@ -97,11 +93,15 @@ class Components extends React.Component<ComponentsGlobalProps, XCSpyState> {
 
     componentDidMount() {
         const props = this.props;
-        // const comps = props.compositionModel.components;
-        const parsers = props.parsers;
         const componentProperties = {};
-        for (let i = 0; i < parsers.length; i++) {
-            const parser = parsers[i];
+        const components = props.compositionModel.components;
+        const parsers = [];
+        components.map((component) => {
+            const parser = new Parser(component);
+            parser.parse();
```

Comment:
Why do you need to define a parsers array ?
You're using the map function in a weird manner. (like a foreach)
basically the map function should transform your component array into a parser array...
Refactor this code